### PR TITLE
Updates README to provide additional guidance on encrypted password process

### PR DESCRIPTION
**Pull request by eemperor**
Changes:
*  Added sample script to verify encrypted password Windows
*  Switch to single quotes in sample Linux bash command to handle special characters.

### CODE REVIEW

**Comment by eemperor** [comment]
@ferricoxide  Is it a particular shell?  The one I'm using handles `!` fine in the middle or beginning of password.

**Comment by eemperor** [comment]
@plus3it/salt-formulas Pinning to older major version to workaround [Salt Issue #54765](https://github.com/saltstack/salt/issues/54765) until a fix is released.

File: .travis.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ install:
         --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
         --name centos-${OS_VERSION} local/centos:${OS_VERSION} init
       sudo docker exec centos-${OS_VERSION} yum -y install \
-        https://repo.saltstack.com/yum/redhat/salt-repo-latest-2.el${OS_VERSION}.noarch.rpm
+        https://repo.saltstack.com/yum/redhat/salt-repo-2018.3-2.el${OS_VERSION}.noarch.rpm
```
Comment (lorengordon) [inline .travis.yml]: maybe expose this as a global env... `SALT_RPM_VERSION=2018.3-2`

Comment (eemperor) [reply .travis.yml]: Global env added

**Pull request merged**